src/pages/index.tsx

```diff
@@ -53,7 +53,9 @@ const MainSection = () => (
       </Heading>
       <Flex py="6" justify="center" align="center">
         <NextLink href="/comparator">
-          <CTAButton>Try me now!</CTAButton>
+          <Button as="a" variant="cta">
```

Comment:
I'm passing `as="a"` since the CTA here should be a link under the hood. Nothing changes visually for the button tho.